src/components/ProductCarousel/index.tsx

```diff
@@ -4,13 +4,14 @@ import Heart from 'components/shared/Icons/Heart';
 export enum ProductTypes {
   Restored = 'Restored',
   New = 'New',
+  Used = 'Used',
```

Comment:
Oh, you already defined here, why not move this to a type file and re use it across your proyect?